app/src/main/res/values/strings.xml

```diff
@@ -207,73 +209,73 @@
     <string name="pref_name_ui_article_textAlignment_justify">Text alignment: Justify</string>
     <string name="pref_desc_ui_article_textAlignment_justify">Stretches lines to equal width (like in newspapers)</string>
     <string name="pref_name_ui_article_handlePreformattedText">Handling Long Preformatted Text</string>
-    <string name="pref_desc_ui_article_handlePreformattedText">How To handle preformatted text that is too long to show completely.</string>
+    <string name="pref_desc_ui_article_handlePreformattedText">How To handle preformatted text too long to show in full.</string>
     <string-array name="pref_option_handlePreformattedText">
         <item>Scroll inside window (overflow)</item>
         <item>Break lines (pre-wrap)</item>
     </string-array>
     <string name="pref_name_ui_showArticleAddedDialog">Show a dialog when bagging</string>
-    <string name="pref_desc_ui_showArticleAddedDialog">Shows a dialog that allows to favorite/archive/tag a newly added article. A toast message will be displayed instead if the option is disabled.</string>
-    <string name="pref_name_ui_article_fullscreen">Fullscreen Article View</string>
+    <string name="pref_desc_ui_showArticleAddedDialog">Shows a dialog allowing favoriting/archiving/tagging a newly added article, (instead of a toast message).</string>
```

Comment:
```suggestion
    <string name="pref_desc_ui_showArticleAddedDialog">Shows a dialog allowing favoriting/archiving/tagging a newly added article (instead of a toast message).</string>
```